src/quo2/components/settings/data_item/view.cljs

```diff
@@ -70,19 +71,21 @@
       {:size  size
        :blur? blur?
        :theme theme}]
-     [left-subtitle
-      {:theme               theme
-       :size                size
-       :description         description
-       :icon                icon
-       :icon-color          icon-color
-       :blur?               blur?
-       :subtitle            subtitle
-       :customization-color customization-color
-       :emoji               emoji}])])
+     (if custom-description
+       [custom-description]
```

Comment:
what is custom description? we can load in any component here instead? 🤔

Reply:
perhaps `left-subtitle` is a better prop name here? and default it to this internal left-subtitle component. custom-description can then have access to all of these props too
e.g
```
  [{:keys [theme title status size blur? description left-subtitle icon subtitle label icon-color
           customization-color emoji] :or {:left-subtitle left-subtitle-internal}}]
  [rn/view {:style style/left-side}
   ...
     [left-subtitle
      {:theme               theme
       :size                size
       :description         description
       :icon                icon
       :icon-color          icon-color
       :blur?               blur?
       :subtitle            subtitle
       :customization-color customization-color
       :emoji               emoji}])])
```

Reply:
or left-subtitle-render-fn 🤔

Reply:
@J-Son89 yes custom-description will load any component. I renamed it to `custom-subtitle` and passed to it the props object